modules/stanford_people_spotlight_views/stanford_people_spotlight_views.info

```diff
@@ -2,7 +2,7 @@ name = Stanford People Spotlight views
 description = Views for Stanford People Spotlight
 core = 7.x
 package = Stanford
-version = 7.x-2.1-dev
+version = 7.x-2.1-alpha5
```

Comment:
Hey @minorwm,
Unless we've changed our process, I thought we only changed the version number when we tagged the repo

Reply:
Yes, you are correct @cjwest.

This should stay as -dev until a release is punched or the version will span multiple git commits and cause version ambiguity for Drupal and future developers.